source/create_utplsql_owner.sql

```diff
@@ -27,9 +27,10 @@ define ut3_tablespace = &3
 
 create user &ut3_user identified by &ut3_password default tablespace &ut3_tablespace quota unlimited on &ut3_tablespace;
 
-grant create session, create sequence, create procedure, create type, create table, create view, create synonym to &ut3_user;
+grant create session, create sequence, create procedure, create type, create table, create view, create synonym, select any dictionary to &ut3_user;
```

Comment:
or not grant the privilege at all.. and use `AUTHID CURRENT_USER` and required that the user running the test has the correct permissions .